sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/Utils.java

```diff
@@ -0,0 +1,228 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.containers.containerregistry;
+
+import com.azure.containers.containerregistry.implementation.models.AcrErrorsException;
+import com.azure.containers.containerregistry.implementation.models.ManifestAttributesBase;
+import com.azure.containers.containerregistry.implementation.models.ManifestAttributesManifestReferences;
+import com.azure.containers.containerregistry.models.TagProperties;
+import com.azure.containers.containerregistry.models.ContentProperties;
+import com.azure.containers.containerregistry.models.RegistryArtifactProperties;
+import com.azure.containers.containerregistry.models.RepositoryProperties;
+import com.azure.containers.containerregistry.models.DeleteRepositoryResult;
+import com.azure.core.exception.ClientAuthenticationException;
+import com.azure.core.exception.HttpResponseException;
+import com.azure.core.exception.ResourceExistsException;
+import com.azure.core.exception.ResourceModifiedException;
+import com.azure.core.exception.ResourceNotFoundException;
+import com.azure.core.http.HttpHeaders;
+import com.azure.core.http.HttpResponse;
+import com.azure.core.http.rest.PagedResponse;
+import com.azure.core.http.rest.PagedResponseBase;
+import com.azure.core.http.rest.Response;
+import com.azure.core.http.rest.ResponseBase;
+import com.azure.core.util.CoreUtils;
+
+import java.util.List;
+import java.util.function.Function;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+final class Utils {
+    private static final String CONTINUATIONLINK_HEADER_NAME = "Link";
+    private static final String LINK_MATCHER = "(<(.+)>;.*)";
```

Comment:
Can make the outer capture group a non-capturing group as I don't see it being used.

```suggestion
    private static final String LINK_MATCHER = "(?:<(.+)>;.*)";
```

Will need to update locations using groups to decrement one